pegasus/test/test_dev_routes.rb

```diff
@@ -199,7 +201,10 @@ def make_test_pegasus
       [:development, :staging, :adhoc, :levelbuilder, :production].each do |env|
         in_rack_env(env) do
           pegasus = make_test_pegasus
-          pegasus.post '/api/dev/set-last-dtt-green', @default_params
```

Comment:
It looks like the original problem might have just been default_param vs default_param**s** here?